ios/RCCNavigationController.m

```diff
@@ -406,13 +406,16 @@ -(void)setButtons:(NSArray*)buttons viewController:(UIViewController*)viewContro
     }
   }
   
+  UIBarButtonItem *itemToIncreaseInsets = [barButtonItems lastObject];
```

Comment:
Why does this call `lastObject`? I don't understand why we only need to change insets for the last item.

Reply:
In other case the previous ones insets may overlap with the next ones